skyportal/handlers/api/thumbnail.py

```diff
@@ -253,3 +273,345 @@ def delete(self, thumbnail_id):
             session.commit()
 
             return self.success()
+
+
+class ThumbnailPathHandler(BaseHandler):
+    @permissions(['System admin'])
+    def get(self):
+        """
+        ---
+        description: |
+          Get information on thumbnails that are
+          or are not in the correct folder/path.
+        tags:
+          - thumbnails
+        parameters:
+          - in: query
+            name: types
+            required: false
+            default: ['new', 'ref', 'sub']
```

Comment:
Maybe the default should be all allowed types?

Reply:
The default is all types that we save on disk locally...

Reply:
Ah I didn't think about that some thumbnails are not local.